custodia/secrets.py

```diff
@@ -10,17 +10,15 @@
 from custodia.message.common import UnknownMessageType
 from custodia.message.formats import Validator
 from custodia.plugin import HTTPConsumer, HTTPError
-from custodia.plugin import CSStoreError, CSStoreExists
+from custodia.plugin import CSStoreError, CSStoreExists, PluginOption
```

Comment:
pep8py2 doesn't complain on my machine although I have the import order checker installed. Maybe you have a different version than 0.9.2? Your versions picks up the problem correctly (C > H > P).